src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -912,6 +915,7 @@ protected BoundExpression BindInferredVariableInitializer(DiagnosticBag diagnost
             TypeSymbolWithAnnotations declTypeOpt,
             AliasSymbol aliasOpt,
             DiagnosticBag diagnostics,
+            bool bindType,
```

Comment:
>bindType [](start = 17, length = 8)

The name of the parameter is somewhat misleading. The type syntax has already been bound. Consider renaming into "createBoundType" or "includeBoundType", or something else that reflects the purpose more accurate.

Reply:
Where has the type syntax been bound?

Reply:
>Where has the type syntax been bound?

For example, on the line 648 in this file.
```
TypeSymbolWithAnnotations declType = BindVariableType(node.Declaration, diagnostics, typeSyntax, ref isConst, isVar: out isVar, alias: out alias);
```

---
In reply to: [255649364](https://github.com/dotnet/roslyn/pull/32843#discussion_r255649364) [](ancestors = 255649364)

Reply:
That's not actually a boundNode, so is it correct to call that binding a type?

Reply:
> That's not actually a boundNode, so is it correct to call that binding a type?

Binding term doesn't necessary imply creating bound nodes. Where else do we bind the type syntax, i.e. figuring what type the type syntax is referencing?

---
In reply to: [257821773](https://github.com/dotnet/roslyn/pull/32843#discussion_r257821773) [](ancestors = 257821773)